deploy-board/deploy_board/webapp/helpers/settings_logging.py

```diff
@@ -0,0 +1,37 @@
+import json
+import logging
+
+
+class StructuredMessage(logging.Formatter):
```

Comment:
It might be possible to instantiate a formatter without inheritance. Do you want to meet up and discuss soon? https://docs.python.org/3/library/logging.html#logging.Formatter